### ISSUE TITLE
hotio.dev support

### ISSUE THREAD

**Issue by alexdelprete** [fmartinou/whats-up-docker#148]
Hi,

could [hotio.dev](https://[hotio.dev](https://hotio.dev/)/) be supported as registry?

Thanks. :)

**Comment by fmartinou** [comment]
Hi,

It's implemented in version `5.19.0`.
The [`hotio` registry](https://fmartinou.github.io/whats-up-docker/#/configuration/registries/hotio/) is enabled by default so it should work out-of-the-box.

I did some tests with the following container:
```yaml
version: "3"
services:
  hotio_radarr:
    image: cr.hotio.dev/hotio/radarr:release-4.0.4.5922
    labels:
      - wud.tag.include=^release-\d+\.\d+.\d+.\d+$$
```

for which wud found a `release-4.1.0.6175` candidate... so far, so good ;)

**Comment by alexdelprete** [comment]
wow...that was fast. :)

i guess the issue with lscr.io is a harder task. thanks a lot for this one.
